src/libertem/analysis/masks.py

```diff
@@ -16,17 +16,25 @@ def dtype(self):
 
     def get_job(self):
         mask_factories = self.get_mask_factories()
-        job = ApplyMasksJob(dataset=self.dataset, mask_factories=mask_factories)
+        use_sparse = self.get_use_sparse()
+        job = ApplyMasksJob(dataset=self.dataset,
+                mask_factories=mask_factories, use_sparse=use_sparse)
         return job
 
     def get_mask_factories(self):
         raise NotImplementedError()
 
+    def get_use_sparse(self):
```

Comment:
As discussed, default impl which returns `False` would be nice, so subclasses can opt-in to sparse matrices.

Reply:
Done! :-)